project/Kang-Ding-Xin-NYCU/README.rst

```diff
@@ -0,0 +1,54 @@
+Basic Information
+=================
+
+PolygonAreaCalaulator
+
+`Kang-Ding-Xin-NYCU <https://github.com/Kang-Ding-Xin-NYCU>`__ for
+`PolygonAreaCalaulator <https://github.com/Kang-Ding-Xin-NYCU/PolygonAreaCaculator/README.rst>`__
+:https://github.com/Kang-Ding-Xin-NYCU/PolygonAreaCaculator
+
+Problem to Solve
+================
+
+* Easily get the the area of polygons and the image of polygons
```

Comment:
Please specify how the code handles holes in polygons and nested holes.  (There are basically two styles to represent holes: cutlines and islands.)